sdk/samples/storage/blobs/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@ if(TRANSPORT_CURL)
   endif()
 endif()
 
+# Generate map file
+if(MSVC)
+    add_link_options(/MAP)
+elseif(CMAKE_C_COMPILER_ID MATCHES "Clang")
+    add_link_options(-Wl,-map,blobs_client_example.map)
+    add_link_options(-Os)
+else()
+    add_link_options(-Xlinker -Map=blobs_client_example.map)
+    add_link_options(-Os)
+endif()
+
```

Comment:
nit: We should move this to a cmake module, like we discussed earlier. Unlike the target name conversation, I don't think it makes sense to repeat a lot o this logic in all test and sample `CMakeLists.txt`.

Btw, we included this in the storage samples but not the IoT ones. Why?